### PR TITLE
New head block

### LINKED ISSUE #36: [Enhancement]: New `head` block
### My idea

- Create new block called `head` in `base.html`
- Move star background script to `index.html` (inside new block)

### PR DESCRIPTION
**Pull request by Jachu8**
issue #36

### CODE REVIEW

File: core/templates/core/base.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
+        {% block head %}
```
Comment (Kubaryt) [inline core/templates/core/base.html:4]:
```diff
        -{% block head %}
```

File: core/templates/core/base.html [hunk 2]
```diff
@@ -12,6 +13,7 @@
         <link href="{% static 'fontawesomefree/css/fontawesome.css' %}" rel="stylesheet" type="text/css">
         <link href="{% static 'fontawesomefree/css/brands.css' %}" rel="stylesheet" type="text/css">
         <link href="{% static 'fontawesomefree/css/solid.css' %}" rel="stylesheet" type="text/css">
+        {% endblock head %}
```
Comment (Sanielia) [inline core/templates/core/base.html]: move {% block head %} to line 15, delete this `head` from endblock

File: core/templates/core/index.html [hunk 3]
```diff
@@ -2,6 +2,27 @@
...
+        randomOpacity: true,
+    });
+    function resizeCanvas() {
+        canvas.width = window.innerWidth;
+        canvas.height = window.innerHeight;
+    }
+
+    window.addEventListener('resize', resizeCanvas);
+</script>
+{% endblock head %}
```
Comment (Sanielia) [inline core/templates/core/index.html]:
```suggestion
{% endblock %}
```

**Review by Sanielia — CHANGES_REQUESTED** [review]
Name branches in this schema: <resolved issue type>/<resolved issue name>, so in this case enhancement/head-block.
Name pull requests in this schema: <resolved issue type>: <what you did in this branch>, so in this case enhancement: Created `head` block
Name commits in this schema: <commit type>: <what you did in this commit>, so in this example enhancement: Created `head` block in `base.html`

**Review by Sanielia — CHANGES_REQUESTED** [review]
just do https://github.com/Gwardia-Czapli/GwardiaHub/pull/39/files#r1567826489

File: core/templates/core/index.html [hunk 4]
```diff
@@ -2,6 +2,27 @@
 
 {% block title %}Strona Główna{% endblock %}
 
+{% block head %}
+<script>
```
Comment (Norbiros) [inline core/templates/core/index.html]:
```suggestion
<script type="module">
```

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: core/templates/core/index.html (hunk 4 above)
Comment (Sanielia) [inline core/templates/core/index.html]:
```suggestion
<script type="module">
```

File: core/templates/core/base.html [hunk 5]
```diff
@@ -12,6 +12,8 @@
         <link href="{% static 'fontawesomefree/css/fontawesome.css' %}" rel="stylesheet" type="text/css">
         <link href="{% static 'fontawesomefree/css/brands.css' %}" rel="stylesheet" type="text/css">
         <link href="{% static 'fontawesomefree/css/solid.css' %}" rel="stylesheet" type="text/css">
+        {% block head %}
+        {% endblock %}
     </head>
     <body class="bg-main_bg text-white">
```
Comment (Sanielia) [inline core/templates/core/base.html:18]:
```suggestion
    <body class="bg-black text-white">
```

**Review by Sanielia — CHANGES_REQUESTED** [review]

**Review by Sanielia — APPROVED** [review]
UwU

**Review by goteusz-maszyk — APPROVED** [review]

**Review by tapczanek1321 — APPROVED** [review]

**Review by Norbiros — APPROVED** [review]

**Pull request merged**